### PR TITLE
add DOI citation badge

### PR DESCRIPTION
**Pull request by mgovers**
cfr https://github.com/PowerGridModel/power-grid-model/pull/275

preview readme: https://github.com/PowerGridModel/power-grid-model-io/tree/feature/citations
preview release DOI: https://zenodo.org/record/8062704
preview docs: https://power-grid-model-io--187.org.readthedocs.build/en/187/citations/CITATION.html

### CODE REVIEW

**Comment by mgovers** [comment]
@TonyXiang8787 this is also ready

File: docs/contribution/folder-structure.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
```
Comment (TonyXiang8787) [inline docs/contribution/folder-structure.md:1]: This file is not correct for this repository. I don't think you need this file and this document entry for pgm-io. The folder structure is pretty simple.

File: docs/index.md [hunk 2]
```diff
@@ -15,6 +15,11 @@ SPDX-License-Identifier: MPL-2.0
 Power Grid Model IO is a tool to convert grid data to and from the native data format of [power-grid-model](https://github.com/PowerGridModel/power-grid-model).
 Currently, conversions from Vision excel exports is possible. Pandapower conversions are under development.
 
+## Citations
+
+If you are using Power Grid Model IO in your research work, please consider citing our library using the references in [Citation](citations/CITATION.md)
```
Comment (TonyXiang8787) [inline docs/index.md:20]: I think in repo PGM you actually download citation.md from .github, and then refer it in the index. Why do you do different here?

Comment (mgovers) [reply docs/index.md]: whoops forgot to push

File: docs/contribution/folder-structure.md (hunk 1 above)
Comment (mgovers) [reply docs/contribution/folder-structure.md]: this happens when you multitask too much 😬 i thought i deleted this. i will go over this PR once again to check whether everything is correct

**Comment by mgovers** [comment]
alright, should be good to go now

**Review by TonyXiang8787 — APPROVED** [review]

**Pull request merged**